projects/packages/videopress/src/client/admin/components/video-filter/style.module.scss

```diff
@@ -59,3 +59,9 @@
 		}
 	}
 }
+
+.title-adornment {
+	fill: var( --jp-gray-10 );
+	margin-left: var( --spacing-base );
+	display: inline-flex;
+}
```

Comment:
The stretches too much IMO:
<img width="810" alt="image" src="https://user-images.githubusercontent.com/77539/221969164-1033a36a-e661-4d19-97ec-e8c8fa24497d.png">

We can reduce the width by applying the following suggestion:

```suggestion
.title-adornment {
	fill: var( --jp-gray-10 );
	margin-left: var( --spacing-base );
	display: inline-flex;

	:global( .components-tooltip.components-popover .components-popover__content ) {
		min-width: auto;
		max-width: 256px;
		width: 256px;
		white-space: break-spaces;
	}
}
```

<img width="594" alt="image" src="https://user-images.githubusercontent.com/77539/221969392-b1fcff89-0adf-4db5-82d4-627acc596960.png">

Reply:
Way more beautiful, indeed!